gateway/1.5/modules/ROOT/pages/policies-pdk-develop-custom-policies.adoc

```diff
@@ -2,4 +2,86 @@
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:imagesdir: ../assets/images
\ No newline at end of file
+:imagesdir: ../assets/images
+
+After installing all the xref:policies-pdk-prerequisites.adoc[], you can begin developing custom policies with the Policy Development Kit (PDK).
+
+The custom policy lifecycle consists of the following docs that explain all steps of developing a custom policy that occur in PDK. The following docs do not include steps that occur outside of PDK, such as xref:policies-pdk-deploy-policies.adoc[applying a policy].
+
+After you xref:policies-pdk-create-project.adoc[Create a New Project], you might iteratively complete some steps. For example, while writing the policy's source code, you may need to return to xref:policies-pdk-create-schema-definition.adoc[Define Policy Configuration Parameters] to add additional parameters, or you might need to recompile the policy after debugging the source code.
```

Comment:
This reads as "After you Create a New Project." It should be "After you create a new project," so the link should be in lower case.